### PR TITLE
Flag phone numbers for reuse

### PR DESCRIPTION
**Pull request by bytheway875**
Branches off of flagging bank account for reuse.

### CODE REVIEW

File: spec/services/fraud_indicator_service_spec.rb [hunk 1]
```diff
@@ -155,5 +155,46 @@
...
+      end
+
+      context "when there are fewer than 3 CTC intakes with the same phone number" do
+        let(:intake) { create :ctc_intake, phone_number: "+15124441234" }
+
+        before do
+          create :ctc_intake, phone_number: "+15124441234"
+        end
+
+        it "marks for fraud" do
```
Comment (jenny-heath) [inline spec/services/fraud_indicator_service_spec.rb:180]: should say "doesn't mark for fraud"

File: spec/services/fraud_indicator_service_spec.rb [hunk 2]
```diff
@@ -155,5 +155,46 @@
...
+      context "when there are GYR intakes with the same phone number" do
+        let(:intake) { create :ctc_intake, phone_number: "+18324658840" }
+
+        before do
+          3.times do
+            create :intake, phone_number: "+18324758840", type: "Intake::GyrIntake"
+          end
+        end
+
+        it "marks for fraud" do
```
Comment (jenny-heath) [inline spec/services/fraud_indicator_service_spec.rb:194]: ditto

**Review by jenny-heath — APPROVED** [review]
looks good! nit: a couple of spec descriptions that say the opposite of what they mean. also i'm not familiar with the DeduplificationService but assuming that was reviewed earlier today (the [PR](https://github.com/codeforamerica/vita-min/pull/1763) doesn't indicate a pair or review but these things don't always get documented)

**Comment by bytheway875** [comment]
@jenny-heath good catch! fixed -- yes, had been working on that with asheesh. we tuple reviewed and revised.

**Pull request merged**